src/lib/addons/datadog.ts

```diff
@@ -58,6 +84,45 @@ export default class DatadogAddon extends Addon {
         );
     }
 
+    generateEnvironmentToggleText(event: IEvent): string {
+        const { environment, project, data, type } = event;
+        const toggleStatus =
+            type === FEATURE_ENVIRONMENT_ENABLED ? 'enabled' : 'disabled';
+        const feature = `<${this.featureLink(event)}|${data.name}>`;
+        return `The feature toggle *${feature}* in the ${project} project had the environment ${environment} ${toggleStatus}`;
```

Comment:
What about: 

`The feature toggle *${feature}* in the ${project} project ${toggleStatus} in *${environment}*`?

Reply:
The feature toggle *${feature}* in the ${project} project was ${toggleStatus} in *${environment}*

Reply:
Ah, never mind, I like yours better